jujugui/static/gui/src/app/components/inset-select/inset-select.js

```diff
@@ -93,12 +93,24 @@ YUI.add('inset-select', function() {
     */
     _generateOptions: function() {
       return this.props.options.map((option, i) => {
-        return (
-          <option
-            key={option.value + i}
-            value={option.value}>
-            {option.label}
-          </option>);
+        let markup;
+        if (option.selected) {
+          markup = (
+            <option key={option.value + i}
+              selected="selected"
```

Comment:
Instead of duplicating the element would it be possible to do something like `selected={option.selected ? 'selected' : undefined}`?